scripts/uosc.lua

```diff
@@ -3129,7 +3131,7 @@ function Controls:serialize()
 		for _, disposition in ipairs(dispositions) do
 			local value = disposition:sub(1, 1) ~= '!'
 			local name = not value and disposition:sub(2) or disposition
-			local prop = (name == 'has_audio' or name == 'has_playlist') and name or 'is_' .. name
+			local prop = (name == 'has_audio' or name == 'has_sub' or name == 'has_chapter' or name == 'has_playlist') and name or 'is_' .. name
```

Comment:
This is over the 120 char limit.

I'd format it like this:
``` Lua
local prop = (name == 'has_audio' or
              name == 'has_sub' or
              name == 'has_chapter' or
              name == 'has_playlist') and name or 'is_' .. name
```

Alternatively this is slowly approaching a point where something like this starts to make sense:
``` Lua
local has_props = {'has_audio', 'has_playlist', 'has_chapter', 'has_sub'}
local prop = itable_index_of(has_props, name) and name or 'is_' .. name
```
But that has some runtime overhead. Don't know which @tomasklaen would prefer.

Reply:
```lua
name:sub(1, 4) == 'has_' ? name or 'is_' .. name
```